### DISCUSSION TITLE
halo2.20 里面的菜单如何添加图标？

### DISCUSSION THREAD

**Discussion by CloudWise-Lukemiao** [halo-dev/halo#7086] [Q&A]
### 前置条件

- [x] 已经在 [issues](https://github.com/halo-dev/halo/issues) 列表中搜索了相关问题。
- [x] 这是 Halo 项目本身存在的问题，如果是非项目本身的问题（如：安装部署问题），建议在 [Discussions](https://github.com/halo-dev/halo/discussions) 提交。
- [x] 已经尝试过停用所有的插件，排除是插件导致的问题。
- [x] 如果是插件和主题的问题，请在对应的插件和主题仓库提交。

### 系统信息

halo 2.20版本，mysql 做主题开发

### 使用的哪种方式运行？

Docker

### 发生了什么？

halo2.20 里面的进行主题开发，菜单如何添加图标，为什么找不到呢？还有一个问题是多个菜单使用统一个自定义模版，该如何把菜单类别或者名称传递过去给模版进而获取对应的数据？

### 相关日志输出

```shell

```

**Comment by ruibaby** [comment]
可参考：

- https://docs.halo.run/developer-guide/annotations-form
- https://docs.halo.run/developer-guide/theme/annotations
